grpc/src/test/proto/testing/grpc/transcoding.proto

```diff
@@ -237,6 +237,12 @@ service HttpJsonTranscodingTestService {
       post: "/v1/arbitrary_wrapped"
     };
   }
+
+  rpc EchoBidirectionalStream(stream Message) returns (stream Message) {
```

Comment:
I have verified the tests used to fail before the fix on `HttpTranscodingService.java`.

In my opinion we shouldn't be so harsh about unsupported features and punish developers.